HW7/src/com/jvprogram/atomic/AtomicApp.java

```diff
@@ -0,0 +1,14 @@
+package com.jv_program.atomic;
```

Comment:
https://mate-academy.github.io/style-guides/java/java.html#s5.2.1-package-names
Package names are all lowercase, with consecutive words simply concatenated together (no underscores). For example, com.example.deepspace, not com.example.deepSpace or com.example.deep_space.